assets/app/scripts/constants.js

```diff
@@ -15,6 +15,7 @@ window.OPENSHIFT_CONSTANTS = {
     "route-types":             "https://docs.openshift.org/latest/architecture/core_concepts/routes.html#route-types",
     "persistent_volumes":      "https://docs.openshift.org/latest/dev_guide/persistent_volumes.html",
     "compute_resources":       "https://docs.openshift.org/latest/dev_guide/compute_resources.html",
+    "pod_autoscaling":         "https://docs.openshift.org/latest/dev_guide/pod_autoscaling.html",
```

Comment:
you'll want to update extensions/examples/online-extensions.js, we should probably add a comment above this section that says to do that.